### PR TITLE
Add default component name for adaptors (#2569)

### LINKED ISSUE #2569: Default component name for adaptors
Flow to potential adaptors (and potential to flow) have different settings concerning default component name. In particular:
- `GeneralCurrentToVoltageAdaptor` and `GeneralVoltageToCurrentAdaptor` have no defaut names,
- `GeneralAngleToTorqueAdaptor` and `GeneralTorqueToAngleAdaptor` have default names omitting "general" prefix,
- `GeneralPositionToForceAdaptor` and `GeneralForceToPositionAdaptor` have no defaut names,
- `HeatflowToTemperatureAdaptor` and `TemperatureToHeatflowAdaptor` have default names "adaptor".

This should be harmonised. I prefer the version which omits "general" prefix in component name.

### LINKED ISSUE #2653: Delete superfluous prefix "wheel" from parameters' name
Remove prefix "wheel" from parameters' name of `Mechanics.MultiBody.Parts.RollingWheel` since actually all parameters concern the wheel. In particular:
- wheelRadius -> radius
- wheelMass -> mass
- wheel_I_axis -> I_axis
- wheel_I_long -> I_long
- wheelColor -> color

### PR DESCRIPTION
**Pull request by AHaumer**
Adaptor issues resolved

close #2569

### CODE REVIEW

File: Modelica/Electrical/Analog/Basic.mo [hunk 1]
```diff
@@ -1888,7 +1888,7 @@ Bear in mind that separating physical components and connecting them via adaptor
     pin_p.i + pin_n.i = 0;
     y = i "output = flow = current";
     u = v "input = potential = voltage";
-    annotation (Documentation(info="<html>
+    annotation (defaultComponentName="coltageToCurrentAdaptor", Documentation(info="<html>
```
Comment (tobolar) [inline Modelica/Electrical/Analog/Basic.mo]: Typo: should be voltageToCurrentAdaptor

**Comment by dietmarw** [comment]
I guess the library officers of Electrical.Analog should really review this.

**Comment by AHaumer** [comment]
correctd the typo.
Sorry for Thermal.HatFransfer, I did the changes on my PC but something went wrong during commit/push. It's corrected now.

**Comment by dietmarw** [comment]
@christiankral Thanks. It needs to be strict since now the merge mechanism is depending on the reviews. If a small detail is missed (like that typo) a new PR would have to be created and this we should try to avoid. I hope you understand.

**Comment by AHaumer** [comment]
sorry I wasn't aware that we have a second example using the adaptors.
Resolved by 04359be..3b54ce7 (my last commit to my fork)

**Comment by AHaumer** [comment]
@christiankral BTW AFAIK you can push changes to my fork (should be the same for the release notes) that are then merged when the pull request is accepted

**Comment by tobolar** [comment]
So could we merge now or wait a day for review of @StephanZiegler or @kristinmajetta?
> I guess the library officers of Electrical.Analog should really review this.

**Comment by StephanZiegler** [comment]
Just a minor note from my side (again potential nitpicking): 

Within MSL the nameing "HeatFlow" (capital F, CamelCase) is used (e.r. for package name "Modelica.Thermal.FluidHeatFlow" or SIUnit "Modelica.SIunits.HeatFlowRate" but the adapters use "Heatflow" (minor f).

Not sure, if this qualifies for another change request.

**Comment by dietmarw** [comment]
That is a good point and this really should be fixed now before we have to create a conversion script entry for the next release after this one.

**Comment by StephanZiegler** [comment]
Ok, i'll prepare something and hopefully push to the right rmeote this time...

**Comment by StephanZiegler** [comment]
Can / should I approve my own changes?

**Pull request merged**